Gemfile

```diff
@@ -67,7 +67,7 @@ group :development, :test do
   gem 'pry-byebug'
   gem 'puma'
   gem 'rspec-rails', '~> 4.0', '>= 4.0.1'
-  gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', branch: 'master'
+  gem 'saml_idp', git: 'https://github.com/18F/saml_idp.git', branch: 'main'
```

Comment:
for the future we should point this at a tag... looks like the Gemfile.lock points at version 11, but we only have versions 10 and 12 tagged 🤦 

```
0.10.0.pre.18f
0.12.0.18f
```

Reply:
> for the future we should point this at a tag...

Just to clarify, do you mean "[in] the future", or "for the future['s sake, we should update this now]" ?

Reply:
In the future--- unless you want to go check out the old SHA and push a tag for it

Reply:
There's a tag `0.11.0.18f` that exists in that repository:

https://github.com/18F/saml_idp/tree/0.11.0.18f

I think it may have been tagged on the branch of https://github.com/18F/saml_idp/pull/34 prior to the squash-n-merge to master, but appears to be effectively the same.